templates/artist/artist_detail.html

```diff
@@ -171,11 +184,18 @@ <h1 class="header-font">{{ artist.name }}</h1>
             <hr>
         </div>
         <!-- Current Investors -->
-        <h2>Current Investors</h2>
-        <ul>
+        <ul class="vertical menu" data-accordion-menu>
+          <li>
+        <a href= "#"><h2>CURRENT INVESTORS</h2></a>
+            <ul class="menu vertical nested is-active">
             {% for investor in investors %}
-                <li>{% if investor.public_profile_url %}<a href="{{ investor.public_profile_url }}"><img src="{{ investor.avatar_url }}" alt="{{ investor.name }}" /> {{ investor.name }}</a>{% else %}{{ investor.name }}{% endif %}: ${{ investor.total_investment }}</li>
+                <li>{% if investor.public_profile_url %}
```

Comment:
Thanks for breaking this over several lines, it's definitely a lot easier to read now. Here's how I would prefer the indenting to be:
```
<ul class="vertical menu" data-accordion-menu>
    <li>
        <a href="#"><h2>CURRENT INVESTORS</h2></a>
        <ul class="menu vertical nested is-active">
            {% for investor in investors %}
                <li>
                    {% if investor.public_profile_url %}
                        <a href="{{ investor.public_profile_url }}">
                            <img class="profile-crop" src="{{ investor.avatar_url }}" alt="{{ investor.name }}" />
                            {{ investor.name }}
                        </a>
                    {% else %}
                        {{ investor.name }}
                    {% endif %}
                    : ${{ investor.total_investment }}
                </li>
            {% endfor %}
        </ul>
    </li>
</ul>
```